docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,9 +98,15 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > - HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.
+  > - In case, you consider time efficiency, it is recommend not to create HCE applications and communicate with the Secure Elements provided in the device.
+  > - Alternatively, ensure that the HCE application is always in the launch state.
```

Comment:
I'm not sure with "launch state"..

Hello, @annie-samsung.
Would you please check the "launch state" in this sentence?

Reply:
@jh8801jung 
This sentence is little ambiguous and both @mijin85cho and myself are not satisfied with it. 

I am giving you certain alternatives, if those make sense, kindly replace then: 

**Original Sentence was:** 
 Alternatively, ensure that the HCE application is always in the launching state.

I had changed it to: 
Alternatively, ensure that the HCE application is always in the launch state.

Suggested alternative: 
Alternatively, ensure that the HCE application is always in the ready state.

@mijin85cho  viewpoint was: 

Alternatively, ensure that the HCE application is always in the running state.